x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -272,6 +272,40 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
     })
   );
 
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/job_for_cloning Get job for cloning
+   * @apiName GetJobForCloning
```

Comment:
`GetJobForCloning` needs to be added to `ml/server/routes/apidoc.json` too, so that API docs are generated for this new endpoint.

Reply:
Updated here https://github.com/elastic/kibana/pull/88898/commits/a1fb963cdfbafa67c6df41aa7d1afc8f77b70d28